### PR TITLE
Utility proxy aws

### PR DESCRIPTION
**Pull request by sandrampeter**
## Background

proxy for AWS

- [x] [AWS](https://github.com/hashicorp/terraform-aws-terraform-enterprise/pull/229)
- [x] [Azure](https://github.com/hashicorp/terraform-azurerm-terraform-enterprise/pull/162)
- [x] [GCP](https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/146)
- [x] [ptfe-replicated](https://app.circleci.com/pipelines/github/hashicorp/ptfe-replicated/3449/workflows/9ca2a2b5-dc0f-49c7-af5c-28446fbcb96c)

### CODE REVIEW

File: fixtures/test_proxy_init/templates/mitmproxy.sh.tpl [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
+sleep 30
+
+apt-get --yes --option "Acquire::Retries=5" update
+
+echo "[$(date +"%FT%T")] Starting mitmproxy startup script" | tee --append $log_pathname
+
+echo "[$(date +"%FT%T")] Install JQ" | tee --append $log_pathname
+
 apt-get update -y
 apt-get install -y jq
```
Comment (anniehedgpeth) [inline fixtures/test_proxy_init/templates/mitmproxy.sh.tpl:18]: Where is jq being used?

**Review by anniehedgpeth — COMMENTED** [review]
Looks good, just a quick question about jq.

File: fixtures/test_proxy_init/templates/mitmproxy.sh.tpl (hunk 1 above)
Comment (sandrampeter) [reply fixtures/test_proxy_init/templates/mitmproxy.sh.tpl:18]: for fetching secrets

Comment (anniehedgpeth) [reply fixtures/test_proxy_init/templates/mitmproxy.sh.tpl:18]: Oh right, for mitm - I see that the secrets function is already in `main`. Thanks!

**Review by anniehedgpeth — APPROVED** [review]

**Pull request merged**